region/tourian/main.json

```diff
@@ -1016,15 +1072,24 @@
                   "notable": false,
                   "requires": [
                     {"or": [
-                      "ScrewAttack",
-                      "Ice"
+                      "Ice",
+                      {"and": [
+                        "ScrewAttack",
+                        "canDodgeWhileShooting"
+                      ]},
+                      {"and": [
+                        "canPseudoScrew",
+                        "canPrepareForNextRoom",
+                        "canDodgeWhileShooting"
+                      ]}
                     ]}
                   ]
                 },
                 {
-                  "name": "Harder Evade",
-                  "notable": false,
-                  "requires": ["canMetroidAvoid"]
+                  "name": "Metroid Room 2 Bottom Door Metroid Avoid",
+                  "notable": true,
+                  "requires": ["canMetroidAvoid"],
```

Comment:
does this need `canPrepareForNextRoom`?

Reply:
If you enter the door on the ground, no.  But we do oftentimes jump through doors.

Reply:
I tried entering the room many different ways and they can all avoid the metroids either by shooting them, or spinjumping towards the wall.